build_tools/ci/cpu_comparison/run.py

```diff
@@ -1021,6 +1023,18 @@ def __init__(self):
             )
         )
 
+        self.register(
+            VanillaMatmul(
+                512,
+                512,
+                4096,
```

Comment:
It should be M == 512, N == 4096, K == 512 to correspond to the problem size we're tracking.

Reply:
Please tell me all the sizes we're tracking (sorry I lost the link to the doc again). I'd like to make this the official source of the tracker.

Reply:
M == 512, N == 4096, K == 512
M == 4096, N == 512, K == 512